client/src/components/App.test.jsx

```diff
@@ -1,6 +1,28 @@
-const { testEnvironment } = require("../../../jest.config");
-const App = require("./App.jsx")
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+// const { testEnvironment } = require('../../../jest.config');
 
-/* test('should do something?', () => {
-  expect();
-}); */
\ No newline at end of file
+describe('App rendering status', () => {
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<App />, div);
+  });
+});
+
+describe('Test parseSkus function', () => {
+  it('parses link to array of id', () => {
+    const app = new App();
+    const link= '<a href="[@trackurl LinkID=\'\' LinkName=\'milksunshinetint\' LinkTag=\'pl-p1\' LinkDesc=\'\' Tracked=\'ON\' Encode=\'OFF\' OfferID=\'\' LinkType=\'REDIRECT\']https://www.sephora.com/product/P410176?skuId=2414258&$deep_link=true[/@trackurl]"  target="_blank">';
```

Comment:
small thing: `link =` - think you're missing a space.
also, may be better to use backticks for a string with lots of quotes, to avoid a lot of escaping?